.pylintrc

```diff
@@ -521,6 +521,6 @@ min-public-methods=2
 
 # Exceptions that will emit a warning when being caught. Defaults to
 # "BaseException, Exception".
-overgeneral-exceptions=BaseException,
-                       Exception,
-                       StandardError
+overgeneral-exceptions=builtsins.BaseException,
```

Comment:
Guess that should be `builtins` here, there's a `s` too much here and in the following lines :wink: